packages/interfaces/src/message.ts

```diff
@@ -32,7 +20,7 @@ export interface IMessage {
 }
 
 export interface IMetaSetter {
-  (message: IProtoMessage & { meta: undefined }): Uint8Array;
+  (message: proto_message.WakuMessage & { meta: undefined }): Uint8Array;
```

Comment:
maybe have a `type IProtoMessage = proto_message.WakuMessage` and re-export it. Less changes across the code base and fair to make the type easier to access